src/legacy/ui/public/saved_objects/components/saved_object_finder.tsx

```diff
@@ -248,73 +328,169 @@ class SavedObjectFinder extends React.Component<SavedObjectFinderProps, SavedObj
             data-test-subj="savedObjectFinderSearchInput"
           />
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiFilterGroup>
+            <EuiToolTip position="top" content={this.getCurrentSortTooltip()}>
+              <EuiButtonEmpty
```

Comment:
Please change this sorting option to be just a normal `<EuiFilterButton>` with context menu popover and the options written out.

![sorting](https://user-images.githubusercontent.com/549577/53984548-913fa080-40e7-11e9-945f-40f6083eeb99.png)


Although, is it really "best match" or is it just "unsorted"?

Reply:
Yes, it is really best match